src/main/java/io/papermc/lib/PaperLib.java

```diff
@@ -147,6 +149,36 @@ public static BlockStateSnapshotResult getBlockState(@Nonnull Block block, boole
         return ENVIRONMENT.getBlockState(block, useSnapshot);
     }
 
+    /**
```

Comment:
These API's weren't added to be async. They were added to more efficiently look up a players UUID than loading the entire offline player.

Nothing would prevent you loading the offline player async too, but I'm not sure this lib needs to handle that.

Lets just drop the async and keep them sync, and plugins can call the method async themselves.

Reply:
Hmm, fair. I guess I was just kinda going based off the current naming scheme. Though you can totally load them async it's not necessarily a requirement.